app/services/duplicate_application.rb

```diff
@@ -83,7 +83,8 @@ def duplicate
         change_references_to_not_requested_yet(references_cancelled_at_eoc)
       end
 
-      new_application_form.update!(references_completed: apply_again?)
+      references_completed = apply_again? && new_application_form.application_references.count >= 2
```

Comment:
2 looks like a magic number here. Is there a place where the number is validated that could be referenced?

Reply:
There is `ApplicationForm::REQUIRED_REFERENCES`

Reply:
There is a method on application form that returns this `complete_references_information?`

Reply:
Missed that. I'll update, thanks. 👍